### PR TITLE
Emitter args

### PR DESCRIPTION
**Pull request by sahilsuneja1**
allows emitters to specify optional args in crawler.conf

### CODE REVIEW

File: crawler/crawler.conf [hunk 1]
```diff
@@ -46,3 +46,14 @@
...
     terminate_fprobe = 1
+
+[ emitters ]
+
+    [[ Stdout Emitter ]]
+    arg_from_conf = 1
+    format = csv
+    
+    [[ File Emitter ]]
+    url = file://xx
```
Comment (nadgowdas) [inline crawler/crawler.conf]: This is the default location for csv file emitter ? Can we put some real-path like file:///tmp/crawler-out or something ?

File: crawler/plugins_manager.py [hunk 2]
```diff
@@ -37,22 +38,69 @@ def get_plugins(
...
-        for plugin in all_emitter_plugins:
-            plugin_obj = plugin.plugin_object
+
+    # getting enabled emitter pluggins from crawler.conf file
+    conf_enabled_plugins = []
+    config = config_parser.get_config()
+    if 'enabled_emitter_plugins' in config['general']:
+        conf_enabled_plugins = config['general']['enabled_emitter_plugins']
+        if 'ALL' in conf_enabled_plugins:
+            conf_enabled_plugins = [p for p in config['emitters']]
```
Comment (nadgowdas) [inline crawler/plugins_manager.py]: So now you have to explicitly enable the plugin in crawler.conf before using it ?

Comment (sahilsuneja1) [inline crawler/plugins_manager.py]: No, --url from the CLI works as before. This is for the case where the plugins are also enabled in conf file.

File: crawler/crawler.conf (hunk 1 above)
Comment (sahilsuneja1) [inline crawler/crawler.conf]: done

**Pull request merged**